spec.emu

```diff
@@ -87,23 +119,27 @@
             1. If Type(_options_) is not Object,
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _assertionsObj_ be Completion(Get(_options_, *"assert"*)).
+            1. Let _assertionsObj_ be Completion(Get(_options_, *"with"*)).
             1. IfAbruptRejectPromise(_assertionsObj_, _promiseCapability_).
+            1. [normative-optional-legacy=""] If _assertionsObj_ is *undefined*, then
+                1. Let _assertionsObj_ be Completion(Get(_options_, *"assert"*)).
+                1. IfAbruptRejectPromise(_assertionsObj_, _promiseCapability_).
             1. If _assertionsObj_ is not *undefined*,
               1. If Type(_assertionsObj_) is not Object,
                 1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
                 1. Return _promiseCapability_.[[Promise]].
               1. Let _keys_ be Completion(EnumerableOwnPropertyNames(_assertionsObj_, ~key~)).
               1. IfAbruptRejectPromise(_keys_, _promiseCapability_).
-              1. Let _supportedAssertions_ be ! HostGetSupportedImportAssertions().
               1. For each String _key_ of _keys_,
                 1. Let _value_ be Completion(Get(_assertionsObj_, _key_)).
                 1. IfAbruptRejectPromise(_value_, _promiseCapability_).
                 1. If Type(_value_) is not String, then
                   1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
                   1. Return _promiseCapability_.[[Promise]].
-                1. If _supportedAssertions_ contains _key_, then
-                  1. Append the ImportAssertion Record { [[Key]]: _key_, [[Value]]: _value_ } to _assertions_.
+                1. Append the ImportAssertion Record { [[Key]]: _key_, [[Value]]: _value_ } to _assertions_.
```

Comment:
Why are they still called assertion records?

Reply:
To minimize the diff and make it easier to see what are the changes, I kept the "assertions" name everywhere (otherwise the diff would have affected a much higher portion of the file). Changing it to "attributes" is just an editorial change, and I plan to do it as soon as we have an agreement on semantics and merged the semantic updates.